server/migrations/1653085228243_add-participant-cohort-date-assigned.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = (pgm) => {
+  pgm.addColumns(
+    'cohort_participants',
+    {
+      date_assigned: {
```

Comment:
please maintain consistency with the rest of the DB column names. We normally use the created_at column. Consider this record immutable.